Content.Shared/Storage/Components/SecretStashComponent.cs

```diff
@@ -65,26 +100,26 @@ public sealed partial class SecretStashComponent : Component
     }
 
     /// <summary>
-    /// Simple pry event for prying open a stash door.
+    ///     Simple pry event for prying open a stash door.
     /// </summary>
     [Serializable, NetSerializable]
-    public sealed partial class StashPryDoAfterEvent : SimpleDoAfterEvent
+    public sealed partial class SecretStashPryDoAfterEventToggleIsOpen : SimpleDoAfterEvent
     {
     }
 
     /// <summary>
-    /// Visualizers for handling stash open closed state if stash has door.
+    ///     Visualizers for handling stash open closed state if stash has door.
     /// </summary>
     [Serializable, NetSerializable]
     public enum StashVisuals : byte
     {
-        DoorVisualState,
+        StashVisualState,
     }
 
     [Serializable, NetSerializable]
-    public enum DoorVisualState : byte
+    public enum StashVisualState : byte
```

Comment:
and this is just a bool

Reply:
I'm not quite sure what your getting at with this! This is the way I've done it in other PRs and everything I can find also does it this way. Mind giving some more detail?